Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js

```diff
@@ -120,13 +120,23 @@ window.asyncEditor.editor = {
         },
         submitEditor(nextEditorGroup) {
             const self = this;
-
-            self.api.submitEditor(
-                self.contentId,
-                self.editorGroup,
-                nextEditorGroup,
-                new FormData(self.$refs.editorForm),
-                (success, data) => { self.processApiData(success, data); });
+            const validationEvent = new CustomEvent('validationEvent', {
+                bubbles: true,
+                cancelable: true,
+                detail: { isValid: true }

```

Comment:
Since there is a `cancelable` setting, you can utilize that, so there is no need for this `isValid` property. You can see how to use it [here](https://developer.mozilla.org/en-US/docs/Web/Events/Creating_and_triggering_events#triggering_built-in_events).

I'd add the async editor itself to the `detail` object, though.